### ISSUE TITLE
Does the schema analysis have any limitations?

### ISSUE THREAD

**Issue by jpmckinney** [kindly/flatterer#16]
e.g. `anyOf`, `allOf` ? I figure this can be added to the documentation.

**Comment by kindly** [comment]
Yes it will not work for anyOf, allOf or any recent conditinal JSONSchema properties like `IF`.  This should be documented.
